docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,566 @@
+
+.. jupyter-execute::
+
+    from qiskit_dynamics import Solver
+    
+    # build solver
+    dt = 1/4.5e9
+    
+    solver = Solver(
+        static_hamiltonian=static_ham_full,
+        hamiltonian_operators=[drive_op0, drive_op1, drive_op0, drive_op1],
+        rotating_frame=static_ham_full,
+        hamiltonian_channels=["d0", "d1", "u0", "u1"],
+        channel_carrier_freqs={"d0": v0, "d1": v1, "u0": v1, "u1": v0},
+        dt=dt,
+    )
+
+Next, instantiate the :class:`.DynamicsBackend`. The ``solver`` is used for simulation,
+``subsystem_dims`` indicates how the full system decomposes for measurement data computation, and
+``solver_options`` are consistent options used by :meth:`.Solver.solve` when simulating the
+differential equation.
+
+Note that, to enable the internal automatic jit-compilation, we choose a JAX integration method.
+
+.. jupyter-execute::
+
+    from qiskit_dynamics import DynamicsBackend
+    
+    # Consistent solver option to use throughout notebook
+    solver_options = {"method": "jax_odeint", "atol": 1e-6, "rtol": 1e-8}
+    
+    backend = DynamicsBackend(
+        solver=solver,
+        subsystem_dims=[dim, dim], # for computing measurement data
+        solver_options=solver_options, # to be used every time run is called
+    )
+
+Alternatively to the above, the :meth:`.DynamicsBackend.from_backend` method can be used to build
+the :class:`.DynamicsBackend` from an existing backend. The above model, which was built manually,
+was taken from qubit :math:`0` and :math:`1` of ``almaden``.
+
+3. Simulating pulse schedules on the :class:`.DynamicsBackend`
+--------------------------------------------------------------
+
+With the above backend, we can already simulate a list of pulse schedules. The code below generates
+a list of schedules specifying experiments on qubit :math:`0`. The schedule is chosen to demonstrate
+that the usual instructions work on the :class:`.DynamicsBackend`.
+
+.. jupyter-execute::
+
+    %%time
+    
+    from qiskit import pulse
+    
+    sigma = 128
+    num_samples = 256
+    
+    schedules = []
+    
+    for amp in np.linspace(0., 1., 10):
+        gauss = pulse.library.Gaussian(
+            num_samples, amp, sigma, name="Parametric Gauss"
+        )
+    
+        with pulse.build() as schedule:
+            with pulse.align_right():
+                pulse.play(gauss, pulse.DriveChannel(0))
+                pulse.shift_phase(0.5, pulse.DriveChannel(0))
+                pulse.shift_frequency(0.1, pulse.DriveChannel(0))
+                pulse.play(gauss, pulse.DriveChannel(0))
+                pulse.acquire(duration=1, qubit_or_channel=0, register=pulse.MemorySlot(0))
+            
+        schedules.append(schedule)
+        
+    job = backend.run(schedules, shots=100)
+    result = job.result()
+
+Retrieve the counts for one of the experiments as would be done using the results object from a real
+backend.
+
+.. jupyter-execute::
+
+    result.get_counts(3)
```

Comment:
Yeah I agree, while it's not that meaningful it's still nice to see, especially since part of the point is to highlight that all the usual instructions work.

I've added a cell drawing the schedule with index `3`.